code/game/machinery/firealarm.dm

```diff
@@ -37,6 +37,27 @@ FIRE ALARM
 
 	var/last_time_pulled //used to prevent pulling spam by same persons
 
+/obj/machinery/firealarm/New(location, direction, building)
```

Comment:
Is there any specific reason you use New() here instead of initialize?

Edit: Ah, I see this is just code from below, but could you still add it to Initialize instead?